src/main/scala/com/datastax/sparkstress/SparkCassandraStress.scala

```diff
@@ -140,12 +147,12 @@ object SparkCassandraStress {
 
   def runTask(config:Config)
   {
-
     val sparkConf =
       new SparkConf()
         .setAppName("SparkStress_"+config.testName)
         //Make sure for streaming that the keep_alive is sufficently large
         .set("spark.cassandra.connection.keep_alive_ms", (config.streamingBatchIntervalSeconds*1000*5).toString)
+        .set("spark.streaming.stopGracefullyOnShutdown", "true")
```

Comment:
This tells it to wait for the receivers to clear their buffers before shutting down. Since we use a queued Dstream i'm not sure it will do anything?